goroutine/gid_test.go

```diff
@@ -47,8 +58,9 @@ func goid() int64 {
 	return id
 }
 
+var gotid int64
+
 func BenchmarkGID(b *testing.B) {
-	var gotid int64
```

Comment:
I don't understand this change

Reply:
it gets optimized to 0ns otherwise with tiny go when not doing the map